### PR TITLE
docs: update VS Code debugging options for 1.47

### PR DESCRIPTION
**Pull request by connor4312**
See details in https://github.com/denoland/vscode_deno/issues/12#issuecomment-658957984

### CODE REVIEW

**Comment by connor4312** [comment]
Fyi: Microsoft has an approval process to go through before I can sign the CLA. I've kicked it off, but expect it to take some time.

**Comment by David-Else** [comment]
@connor4312 @ry Any update on this? The new VS Code 1.48 came out today, and the current Deno docs are wrong. This PR is correct and urgently needs approval.

File: docs/tools/debugger.md [hunk 1]
```diff
@@ -83,6 +83,10 @@ Deno can be debugged using VSCode.
 Official support via the plugin is being worked on -
 https://github.com/denoland/vscode_deno/issues/12
 
+**NOTE**: Until the July release of VS Code (1.48), you will need to use the
+[nightly build](https://github.com/microsoft/vscode-js-debug/#nightly-extension)
+of their debugger extension.
+
```
Comment (David-Else) [inline docs/tools/debugger.md:89]: This **NOTE** is now not needed, 1.48 is out.

**Comment by connor4312** [comment]
Thanks for the ping -- the CLA was approved, but for some reason our system didn't send me a notification about it like it normally does/did 🤔

**Pull request merged**